www/simple.php

```diff
@@ -253,9 +253,13 @@ function getScheme() {
 	$hubic_user = $_GET['hubic_user'];
 	$hubic_password = $_GET['hubic_password'];
 
+        if (is_array($clients[$client]) && array_key_exists('autoredirect_uri', $clients[$client])) {
```

Comment:
Yes, it makes sense `isset()` for arrays, really appreciate the info.

For variables, I think it depends, because sometimes you could need to set variable to `NULL` and in that case `isset()` returns false.